.gitignore

```diff
@@ -1,4 +1,6 @@
 .gradle/
+bin/
```

Comment:
Where does Eclispe create `bin/` directories? Can't we just exclude those?

Reply:
Besides, paths created by local tools should be ignored locally ;)

Reply:
Oh, yes, thats good! Removed it, with two other Eclipse files, in [eb2b65a](https://github.com/metafacture/metafacture-fix/pull/179/commits/eb2b65ae6a82ec1bf489dfe441427ece39ab4bf0).

Reply:
Great, thanks.